backend/webapi/Infrastructure/HttpClients/Keycloak/KeycloakApiDefinitions.cs

```diff
@@ -79,6 +79,8 @@ public class UserRepresentation
 
     public void SetCpn(string cpn) => this.SetAttribute("common_provider_number", cpn);
 
+    public void SetEmail(string email) => this.SetAttribute("pidp_email", email);
```

Comment:
I would call this `SetPidpEmail` to further distinguish it from the `Email` property.